package.xml

```diff
@@ -7,6 +7,12 @@
   <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
+
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
Perhaps this is a question we should answer for all the Python packages, but I'm not sure if this should be a `buildtool` or an `export`, rather than just an `exec` depend. I guess maybe it just needs an exec_depend in addition to this depend.

Basically I think we'd need to see how the Python stuff is handled in a cross-compiling situation...

It definitely seems like there should be an exec_depend on python3 though.